### PR TITLE
Add Weiyun.app v2.0.002.20632

### PR DESCRIPTION
**Pull request by chihosin**
*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked that the cask was not already refused in [closed issues].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed

### CODE REVIEW

File: Casks/weiyun.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+cask 'weiyun' do
+  version '2.0.002.20632'
+  sha256 'e219143abb8b5cbf1063a28f72398e8747bdec0cd95ad03369edbb16a8dd9043'
+
+  url 'http://dldir1.qq.com/weiyun/Weiyun_Mac_2.0.002.20632.dmg'
```
Comment (miccal) [inline Casks/weiyun.rb]: You need to add a [comment](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/url.md#when-url-and-homepage-hostnames-differ-add-a-comment) above the `url`.

Please change the `url` to
```
url "http://dldir1.qq.com/weiyun/Weiyun_Mac_#{version}.dmg"
```

File: Casks/weiyun.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  name 'weiyun'
+  homepage 'http://www.weiyun.com'
+
+  app 'Weiyun.app'
+
```
Comment (miccal) [inline Casks/weiyun.rb]: Please remove this trailing whitespace.

File: Casks/weiyun.rb [hunk 3]
```diff
@@ -0,0 +1,16 @@
+cask 'weiyun' do
+  version '2.0.002.20632'
+  sha256 'e219143abb8b5cbf1063a28f72398e8747bdec0cd95ad03369edbb16a8dd9043'
+
+  # dldir1.qq.com/weiyun was verified as official when first introduced to the cask
+  url 'http://dldir1.qq.com/weiyun/Weiyun_Mac_#{version}.dmg'
```
Comment (miccal) [inline Casks/weiyun.rb]: You need to use double quotes for the interpolation to work, like so:
```
url "http://dldir1.qq.com/weiyun/Weiyun_Mac_#{version}.dmg"
```

File: Casks/weiyun.rb [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+  name 'weiyun'
+  homepage 'http://www.weiyun.com'
+
+  app 'Weiyun.app'
+  uninstall quit: 'com.tencent.MacWeiyun'
+  zap delete: [
```
Comment (miccal) [inline Casks/weiyun.rb]: The `app`, `uninstall` and `zap` stanza's need a blank line between them.

**Comment by ChihoSin** [comment]
I think I should learn how to code some ruby 😂

**Pull request merged**